test/broadway_test.exs

```diff
@@ -2465,6 +2476,84 @@ defmodule BroadwayTest do
     end
   end
 
+  describe "starting under a registry using via tuple" do
+    setup do
+      {:ok, _registry} = start_supervised({Registry, keys: :unique, name: MyRegistry})
+      name = via_tuple({UsesRegistry, "1"})
```

Comment:
For simplicity, I would have this simply be:

    via_tuple(:broadway)

We probably don't need a tuple. :)

Reply:
Good call